client/index.jsx

```diff
@@ -7,6 +7,15 @@ var React = require('react');
 
 module.exports = React.createClass({
 	render: function() {
+		var googleMaps = '';
+
+		if (this.props.useGoogleMaps) {
+			googleMaps = [
+				<script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA9PjKau31h9r6FE2LTuLTeb5WWOPy__Yg"></script>,
+				<script dangerouslySetInnerHTML={{__html: this.props.googleMapsBootstrap}}></script>
+			];
```

Comment:
Can we avoid duplicating this somehow? Looks like you can include subcomponents by [nesting their tags](http://facebook.github.io/react/docs/multiple-components.html#children). Maybe define a GoogleMaps component.